ui/guest/src/store/epics/root.ts

```diff
@@ -269,9 +269,9 @@ const epic = combineEpics<GuestStandardAction, GuestStandardAction, GuestState>(
         };
 
         const models = getCachedModels();
-        const modelId = action.payload.record.modelId;
-        const parentModelId = getParentModelId(modelId, models, modelHierarchyMap);
-        const path = models[parentModelId ?? modelId].craftercms.path;
+        const dropZone = dragContext.dropZone;
```

Comment:
I tested in all cases (placing detached assets, upload assets, sort components, placing new/existing components) and also edition on content inside a component (like updating an image of a component) and the path to be locked is correct.